nominal/nominal.py

```diff
@@ -339,3 +348,41 @@ def _get_start_end_timestamp_csv_file(
         IntegralNanosecondsUTC(start.to_datetime64().astype(int)),
         IntegralNanosecondsUTC(end.to_datetime64().astype(int)),
     )
+
+def create_checklist(
+    assignee_email: str,
+    title: str,
+    checks: Sequence[CreateCheck],
+    checklist_variables: Sequence[CreateChecklistVariable],
+    default_ref_name: str | None = None,
```

Comment:
what is a default_ref_name?

Reply:
Because the compute representation service does not fetch any actual series on the backend, we refer to series by their channel name and ref name - similar to unresolved variable locators in the checklist service. When the checklist is applied to a run, they end up getting resolved into actual series.

Reply:
I guess the fact that it's called "default" is what's tripping me up? it's not really a default right?

Reply:
It's provided out of utility here - i.e. use this ref name when you go to create the unresolved checklist variable so that you don't have to specify the ref name every time you reference a channel